test/chef-test.js

```diff
@@ -7,29 +7,30 @@ describe("Allocator Contract", function () {
   let topChef;
   let metric;
 
+  let origin;
   let allocationGroup1;
   let allocationGroup2;
-  let vestingContract;
+  let addrs;
 
   beforeEach(async function () {
-    [allocationGroup1, allocationGroup2, vestingContract] = await ethers.getSigners();
+    [origin, allocationGroup1, allocationGroup2, ...addrs] = await ethers.getSigners();
```

Comment:
new convention proposal:  let's do `origin` as the first account because they are the `owner` and will usually have the minted supply.  We shouldn't use this account for "user" tests because they aren't a user.